myia/ir/utils.py

```diff
@@ -232,7 +232,8 @@ def sexp_to_node(sexp, graph, multigraph=False, sub=None):
     elif isinstance(sexp, ANFNode):
         return sexp
     else:
-        return Constant(sexp)
+        node = Constant(sexp)
+    return node
```

Comment:
Not sure what the point of these changes is.

Reply:
Leftovers from something else.